mscore/scoreview.cpp

```diff
@@ -276,6 +281,7 @@ ScoreView::~ScoreView()
     delete _bgPixmap;
     delete _fgPixmap;
     delete shadowNote;
+    delete _scoreTreeView;
```

Comment:
If you give your widget a parent then [Qt will delete it for you](https://doc.qt.io/qt-5/objecttrees.html#overview) and this line would not be necessary.